.github/workflows/aws-ip-json.yaml

```diff
@@ -0,0 +1,22 @@
+name: Download AWS IP ranges
```

Comment:
How will that work if it's part of github action?

Reply:
It's a special github thingy called [github flat data](https://next.github.com/projects/flat-data). It'll download the file we set up and commit it to the repo just like in this commit on my fork https://github.com/flavianmissi/quay/commit/30e3c2e924a6b0c3525405ba7afacd01342f3af4.